### ISSUE TITLE
Error when no CI tasks ran

### ISSUE THREAD

**Issue by bradleyayers** [appthrust/moon-ci-retrospect#2]
In some cases there are no CI tasks to run (no inputs changed), and this happens:

```
Error: ENOENT: no such file or directory, open '/home/runner/work/haohao.how/haohao.how/.moon/cache/ciReport.json'
    at async open (node:internal/fs/promises:636:25)
    at async readFile (node:internal/fs/promises:1246:14)
    at async readFileContent (file:///home/runner/work/_actions/appthrust/moon-ci-retrospect/v0/index.js:83:12)
    at async readCiReport (file:///home/runner/work/_actions/appthrust/moon-ci-retrospect/v0/index.js:35:26)
    at async main (file:///home/runner/work/_actions/appthrust/moon-ci-retrospect/v0/index.js:4:22)
    at async file:///home/runner/work/_actions/appthrust/moon-ci-retrospect/v0/index.js:126:5 {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/runner/work/haohao.how/haohao.how/.moon/cache/ciReport.json'
}
```

**Comment by suin** [comment]
@bradleyayers Thank you for reporting this issue. We'll investigate the problem with the CI task execution when no inputs have changed and get back to you with our findings.

**Comment by suin** [comment]
@bradleyayers We've implemented a fix for this issue in #3. It should now handle cases where no CI tasks are executed.

Could you please test it out and let us know if it resolves the problem? If you encounter any issues, feel free to reach out.

Thanks for bringing this to our attention.
